test/models/playlist_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+class PlaylistTest < ActiveSupport::TestCase
+  test "invalid if name is blank" do
+    playlist = Playlist.new(name: "", energy: 35, danceability: 56, genre: "genre1")
```

Comment:
Ici pour simplifier l'écriture des test tu peux faire une méthode avec des arguments valides et les réutiliser partout.

Exemple avec le premier test:

```suggestion
  def valid_playlist_params
    {
      name: 'name1',
      energy: 35, 
      danceability: 56, 
      genre: "genre1"
    } 
  end
  
  test "invalid if name is blank" do
    playlist = Playlist.new(valid_playlist_params.merge(name: ''))
```